.github/workflows/snapshot-file-change.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           fetch-depth: 0 # OR "2" -> To retrieve the preceding commit.
       - name: 'Set current timestamp'
         run: |
-          echo "CUR_TIMESTAMP=$(date +'%Y-%m-%dT%H:%M:%S')" >> $GITHUB_ENV
+          echo "CUR_TIMESTAMP=$(date +'%Y-%m-%dT%H:%M:%SZ')" >> $GITHUB_ENV
```

Comment:
Not sure the Z here is right, how about change to 
```
echo "CUR_TIMESTAMP=$(date --rfc-3339=seconds | sed 's/ /T/')"
```

Reply:
Will do this together with #94